recipe/meta.yaml

```
@@ -13,34 +13,41 @@ build:
   entry_points:
     - ydata_profiling = ydata_profiling.controller.console:main
     - pandas_profiling = ydata_profiling.controller.console:main
-  noarch: python
-  script: {{ PYTHON }} -m pip install . -vv
+  script: {{ PYTHON }} -m pip install . -vv --no-deps
   number: 0
+  skip: True   #  [py<37]
 
 requirements:
   host:
-    - python >=3.7,<3.11
+    - python
     - pip
+    - setuptools
+    - wheel
   run:
-    - python >=3.7,<3.11
-    - scipy >=1.4.1,<1.10
+    - python
+    - scipy=1.9.3
     - pandas >1.1,<1.6,!=1.4.0
     - matplotlib-base >=3.2,<3.7
     - pydantic >=1.8.1,<1.11
     - pyyaml >=5.0.0,<6.1
     - jinja2 >=2.11.1,<3.2
     - visions ==0.7.5
     - numpy >=1.16.0,<1.24
-    - imagehash
-    - pillow
+    # Related to HTML report
     - htmlmin ==0.1.12
+    # Correlations
     - phik >=0.11.1,<0.13
+    # Examples
     - requests >=2.24.0,<2.29
+    # Progress bar
     - tqdm >=4.48.2,<4.65
-    - seaborn-base >=0.10.1,<0.13
+    - seaborn >=0.10.1,<0.13
     - multimethod >=1.4,<1.10
+    # metrics
     - statsmodels >=0.13.2,<0.14
     - typeguard >=2.13.2,<2.14
+    - imagehash
```

```suggestion
    - imagehash ==4.3.1
```
See https://github.com/ydataai/ydata-profiling/blob/v4.1.1/requirements.txt#L24

---

Thanks!